LongWarOfTheChosen/Config/XComEngine.ini

```diff
@@ -15,6 +15,7 @@
 +ModEditPackages=LW_AlienPack_Integrated
 +ModEditPackages=LW_Toolbox_Integrated
 +ModEditPackages=LW_Overhaul
++ModEditPackages=WotCWeaponsAndArmor
```

Comment:
I'll put the commend here. Could you rename this? Probably to `LW_WeaponsAndArmor`. Just for consistency. Plus I think I'd like to refactor other weapon and armour code into this package at some point.

Also, when you do use WOTC, please use all caps for consistency throughout the project. Plus, it's much easier on my fingers to type all caps or all lower case :smile:

Reply:
OK I will rename this. I will rename it to LWOTC_WeaponsAndArmor. Does that work?

Reply:
I still prefer just the LW_ prefix for consistency unless you're planning to release as a separate mod? I'm ok with that, just want to check.

Reply:
ok sure, I'll use LW_instead.

Reply:
This has been changed to LW_WeaponsAndArmor.